Runner/Sandbox.cs

```diff
@@ -194,9 +195,9 @@ public void CloseExectionScope()
             _classInstanceManager.CloseScope();
         }
 
-        [DebuggerStepperBoundary]
-        [DebuggerHidden]
-        public ExecutionResult ExecuteHooks(string hookType, IHooksStrategy strategy, IList<string> applicableTags)
+        //[DebuggerStepperBoundary]
```

Comment:
We need these attributes to prevent the user's debugger to step into runner's code.